rocky/tests/objects/test_objects.py

```diff
@@ -424,3 +424,54 @@ def test_ooi_list_export_csv(rf, client_member, mock_organization_view_octopoes)
     assert len(exported_objects) == 152
     assert "observed_at" in exported_objects[0]
     assert "filters" in exported_objects[0]
+
+
+def test_delete_perms_object_list(
```

Comment:
These should be split into individual tests. Or perhaps consider parametrizing this test using `@pytest.mark.parametrize`